server/src/domain/media/media.repository.ts

```diff
@@ -48,6 +48,7 @@ export interface IMediaRepository {
   extractThumbnailFromExif(input: string, output: string): Promise<void>;
   resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void>;
   crop(input: string, options: CropOptions): Promise<Buffer>;
+  extractThumbhash(imagePath: string): Promise<Buffer>;
```

Comment:
I don't think the verb 'extract' is very accurate. It made sense for exif, but in this case a verb like 'generate' is probably more appropriate.

Reply:
Agree. generateThumbhash or getThumbhash sound good to me.